pkg/configdb/configdb.go

```diff
@@ -0,0 +1,77 @@
+package configdb
+
+import (
+	"encoding/json"
+	"log"
+	"os"
+	"path"
+	"sync"
+)
+
+var mu sync.Mutex
```

Comment:
Please remove this and other global variables: https://github.com/uber-go/guide/blob/master/style.md#reduce-scope-of-variables, https://github.com/uber-go/guide/blob/master/style.md#avoid-mutable-globals.

Reply:
@GiedriusS fixed